src/status_im/contexts/wallet/add_address_to_watch/confirm_address/view.cljs

```diff
@@ -15,8 +15,8 @@
 (defn view
   []
   (let [{:keys [address]}  (rf/sub [:get-screen-params])
-        number-of-accounts (count (rf/sub [:profile/wallet-accounts]))
-        account-name       (reagent/atom (i18n/label :t/default-account-name
+        number-of-accounts (count (filter :watch-only? (rf/sub [:wallet/accounts])))
```

Comment:
@J-Son89 should we have subs that returns watch-only accounts like `(rf/sub [:wallet/watch-only-accounts])` and active accounts like `(rf/sub [:wallet/active-accounts])` (or something similar)

Reply:
Yeah makes sense. Can you add these subs here @Rende11 as it’s related to this pr?

Reply:
@J-Son89 Sure

Reply:
Done